File: cpp_solutions/705. Design HashSet.cc
class MyHashSet {
public:
    set<int> seen;
    MyHashSet() {
        seen.clear();
    }
    
    void add(int key) {
        seen.insert(key);
    }
    
    void remove(int key) {
        seen.erase(key);
    }
    
    bool contains(int key) {
        return (seen.find(key)!=seen.end());
    }
};

/**
 * Your MyHashSet object will be instantiated and called as such:
 * MyHashSet* obj = new MyHashSet();
 * obj->add(key);
 * obj->remove(key);
 * bool param_3 = obj->contains(key);
 */

File: python3_solution_set0/705. Design HashSet.py
##ss
class MyHashSet:

    def __init__(self):
        self.dicts = {}
        

    def add(self, key: int) -> None:
        self.dicts[key] = 1
        
        

    def remove(self, key: int) -> None:
        if key in self.dicts:
            self.dicts.pop(key)
        

    def contains(self, key: int) -> bool:
        return key in self.dicts
        


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)
